### PR TITLE
Add rights statement to bulk ingest

### LINKED ISSUE #5751: Add Rights Statement to Bulk Ingest
### Summary

The rights statement needs to be changed to Copyright Not Evaluated for finding aids material the studio's ingesting, but there's no way to do it from bulk ingest.

### Impact 

Would make bulk ingest significantly easier, so they don't have to bulk ingest to a new collection then bulk edit, or ingest one at a time. This was requested by the studio in #digital_studio.

### Priority recommendation

- [x] asap
- [ ] within the next 3 weeks
- [ ] PO will prioritize

## Sudden Priority Justification
_Required if "asap" or "within the next 3 weeks" is checked. Add "Sudden Priority" and "Maintenance/Research" labels_

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #5751 

Screenshot 2023-05-23 at 12 26 16 PM

### CODE REVIEW

**Review by hackartisan — CHANGES_REQUESTED** [review]
Is there a controller test that ensures the resource doesn't have a rights statement value when that parameter isn't sent? I think that would be nice to have.

**Comment by eliotjordan** [comment]
@hackartisan You have to choose a rights statement. It's a required field in the regular new resource form as well as the bulk ingest form. Previously, bulk ingested resources all received a default value.

File: app/views/bulk_ingest/show.html.erb [hunk 1]
```diff
@@ -37,6 +37,13 @@
                 <label for="collections">Collection(s)</label>
                 <%= select_tag "collections", options_for_select(@collections), multiple: true, class: "form-control" %>
               </div>
+              <div class="form-group">
+                <label for="rights-statement">Rights Statement</label>
+                <%= select_tag :rights_statement,
+                  options_from_collection_for_select(ControlledVocabulary.for(:rights_statement).all, "value", "label"),
+                  class: 'form-control rights-statement',
+                  required: false %>
```
Comment (hackartisan) [inline app/views/bulk_ingest/show.html.erb:45]: Make this `required: true`?

**Review by hackartisan — CHANGES_REQUESTED** [review]
Aha, I see. Okay can we make the form require it?

File: app/views/bulk_ingest/show.html.erb (hunk 1 above)
Comment (eliotjordan) [reply app/views/bulk_ingest/show.html.erb:45]: 👍

**Comment by eliotjordan** [comment]
@hackartisan Ready for re-review.

**Review by hackartisan — APPROVED** [review]
Hm. that doesn't seem to make the "required" label show up on the form. but I'm thinking maybe we don't block this PR on that, what do you think @eliotjordan

**Comment by eliotjordan** [comment]
@hackartisan Yeah, I think it's fine to merge.

**Pull request merged**